cpp/include/raft/core/operators.hpp

```diff
@@ -40,6 +40,14 @@ struct identity_op {
   }
 };
 
+struct no_op {
```

Comment:
Nitpick, but I'm not so sure about naming it `no_op`, due to potential confusion with the old `Nop` which has been replaced by `identity_op`.

What about `void_op` / `empty_op`?

Reply:
Sure! is now void_op